usr/share/rear/skel/default/etc/scripts/system-setup-functions.sh

```diff
@@ -0,0 +1,30 @@
+# Use an artificial array to get the kernel command line parameters as array elements
+kernel_command_line=( $( cat /proc/cmdline ) )
+
+function rear_debug() {
+    for kernel_command_line_parameter in "${kernel_command_line[@]}" ; do
+        test "debug" = "$kernel_command_line_parameter" && return 0
+    done
+    return 1
+}
+
+function unattended_recovery() {
+    for kernel_command_line_parameter in "${kernel_command_line[@]}" ; do
+        test "unattended" = "$kernel_command_line_parameter" && return 0
+    done
+    return 1
+}
+
+function automatic_recovery() {
+    # The unattended recovery mode implies automatic recovery (see the implementations below)
+    # so that in unattended mode the automatic recovery code below must not be run
```

Comment:
This comment needs to be adapted because
the automatic recovery code was moved to
/etc/scripts/run-automatic-rear
so that this comments should now read:
```
    # The unattended recovery mode implies automatic recovery
    # (see the implementation in skel/default/etc/scripts/run-automatic-rear)
    # so that in unattended mode the automatic recovery code must not be run

Reply:
good catch!

Reply:
adapted in the last commit.

Reply:
Almost correct:
In the third line of that commit part
the word 'below' should be removed
because there is no longer
automatic recovery code 'below'.

Reply:
Oops! Thanks for checking. Corrected in the last commit.